### DISCUSSION TITLE
Can you change Korean translation link ? 😆

### DISCUSSION THREAD

**Discussion by dancinlife** [tabwrangler/tabwrangler#395] [General]
Hi, Long time no see.
I had an issue about changing my ID.
simple-is-best

now, there id is another user's.
can you change Korean translation link ? 😆
Thank you!

If i learn swift. I have interesting safari version. 😆

🇰🇷 Korean translation by [simple-is-best](https://github.com/simple-is-best)
->
🇰🇷 Korean translation by [dancinlife](https://github.com/dancinlife)

**Comment by ssorallen** [comment]
Hi there, I listed all contributors from Crowdin and linked directly to their Crowdin profiles: https://github.com/tabwrangler/tabwrangler/commit/8d754e26e4135826fef849aa12f095636e9d2011

I will only support links to Crowdin profiles going forward because I have no way of verifying the authenticity or ownership of links other than to Crowdin. I am happy to promote Crowdin profiles though, and thank you again for your contributions.

**Comment by dancinlife** [comment]
Thanks!!!
